### ISSUE TITLE
output.override.query.signal cannot be set to false

### ISSUE THREAD

**Issue by arharvey918** [anymaniax/orval#753]
## What are the steps to reproduce this issue?

1. Create `orval.config.js` with the following contents (note signal is set to `false` to remove the generation of the abort signal provided by query ([doc](https://orval.dev/reference/configuration/output#signal))):
  ```
  module.exports = {
    petstore: {
      output: {
        client: "react-query",
        clean: true,
        target: "src/petstore.ts",
        override: {
          query: {
            signal: false,
          },
        },
      },
      input: "./petstore.yaml",
    },
  };
  ```

2. Run `npx orval --config ./orval.config.js`

```
$ npx orval --config ./orval.config.js
🍻 Start orval v6.11.1 - A swagger client generator for typescript
petstore: Cleaning output folder
🎉 petstore - Your OpenAPI spec has been converted into ready to use orval!
```

## What happens?

The file `src/petstore.ts` is generated, but `signal` is still present in the generated code.

## What were you expecting to happen?

`signal` should not be present in the generated code.

## Any logs, error output, etc?

n/a

## Any other comments?

This is a guess at what's happening: the `normalizeOptions` function [here](https://github.com/anymaniax/orval/blob/ba7ee988557a63c392c6d9bee50bbd7eddfe3b86/packages/orval/src/utils/options.ts#L179) has a block that seems to indicate that the default value of signal is `true` (the doc doesn't currently state a default, but should it?). The value is applied, followed by a spread operator on `normalizeQueryOptions`. That function has a [line](https://github.com/anymaniax/orval/blob/ba7ee988557a63c392c6d9bee50bbd7eddfe3b86/packages/orval/src/utils/options.ts#L386) which sets signal to `true` if it's set, otherwise it sets no value. Since it returns the parent function with no value for signal, it does not override the value of `true` to `false`.

I would be happy to get a PR up for this if that sounds like that's where the issue is.

## What versions are you using?

**Operating System:** macOS Ventura 13.2
**Package Version:** v6.11.1
**Browser Version:** n/a

**Comment by anymaniax** [comment]
hello @arharvey918, will be fix in the next version

**Comment by anymaniax** [comment]
Released with `6.12.0`

**Comment by arharvey918** [comment]
Thank you! I've verified `signal` is no longer present in the generated code when `output.override.query.signal` is set to false as displayed in the original post.
